src/bindgen/parser.rs

```diff
@@ -718,40 +853,32 @@ impl Parse {
             return;
         }
 
-        let loggable_item_name = || {
-            let mut items = Vec::with_capacity(3);
-            items.push(crate_name.to_owned());
-            if let Some(ref self_type) = self_type {
-                items.push(self_type.to_string());
-            }
-            items.push(sig.ident.unraw().to_string());
-            items.join("::")
-        };
+        let loggable_item_name = format!("{}::{}", crate_name, sig.ident);
```

Comment:
The reason this was done was to avoid having to build this string unconditionally...